### DISCUSSION TITLE
Cadbuildr Playground - Built on top of replicad

### DISCUSSION THREAD

**Discussion by clement91190** [sgenoud/replicad#164] [General]
Hi, 

I wanted to share a project i've been working on for a little while and I thought this community would/could be interested. First thanks a lot @sgenoud for the effort building replicad, this is an industry that needs more open-source initiatives !

I heard about replicad about a year ago while working on cadbuildr, my goal is to create a platform to create CAD from code (for me via python). It starts with an open-source "API" python lib to describe the CAD operations, available here : https://github.com/cadbuildr/foundation, and can be agnostic of the CAD kernel/backend. Being able to have a kernel in the browser (as well as run python via webassembly) is super useful and so replicad is used as that backend in this application : https://playground.cadbuildr.com/ ( a little bit like the Replicad Workbench but for python) . I like python because it's quite used by mechanical engineers already and "easy". I would love to have more people open-source/share parts/assemblies via code just like code is shared on github. 

I'm alternating between seeing if I can build a company out of this or just an open-source project or both. If this is something that interests people here, would love to chat or get feedback ! 

Cheers,
Clement

**Comment by sgenoud** [comment]
This looks great!
